modules/500-upmeter/images/webui/src/app/i18n/en.ts

```diff
@@ -117,6 +117,22 @@ const langPack: LangPack = {
             <p>Group result is a combination of probe results with the priority of the worst results.</p>
             `,
     },
+    nginx: {
```

Comment:
Maybe it is better to give this group a more generic name like `ingress` because we [are planning](https://github.com/deckhouse/deckhouse/issues/140) to add more ingress controllers in the future.

Reply:
Resolved, will rename as needed